apps/veil/src/pages/tournament/ui/round/current-voting-results/index.tsx

```diff
@@ -0,0 +1,165 @@
+import { Amount } from '@penumbra-zone/protobuf/penumbra/core/num/v1/num_pb';
+import { TableRow } from './table-row';
+
+const THRESHOLD = 0.05;
+
+const valueView = new ValueView({
+  valueView: {
+    value: {
+      amount: new Amount({ lo: 133700000n }),
+      metadata: new Metadata({
+        base: 'um',
+        display: 'um',
+        denomUnits: [
+          {
+            denom: 'um',
+            exponent: 6,
+          },
+        ],
+        symbol: 'um',
+        penumbraAssetId: { inner: new Uint8Array([1]) },
+        coingeckoId: 'um',
+        images: [],
+        name: 'um',
+        description: 'um',
+      }),
+    },
+    case: 'knownAssetId',
+  },
+});
+
+export const CurrentVotingResults = () => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [page, setPage] = useState(1);
+  const [limit, setLimit] = useState(10);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setIsLoading(false);
+    }, 1000);
+  }, []);
+
+  const totalPages = 100;
+  const totalVotes = 10000;
+
+  const data = [
+    {
+      symbol: 'USDC',
+      imgUrl:
+        'https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.png',
+      votes: 2777,
+      estimatedIncentive: valueView,
+    },
+    {
+      symbol: 'OSMO',
+      imgUrl:
+        'https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.png',
+      votes: 1337,
+      estimatedIncentive: valueView,
+    },
+    {
+      symbol: 'ATOM',
+      imgUrl:
+        'https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.png',
+      votes: 1000,
+      estimatedIncentive: valueView,
+    },
+    {
+      symbol: 'USDC',
+      imgUrl:
+        'https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.png',
+      votes: 7,
+      estimatedIncentive: valueView,
+    },
+    {
+      symbol: 'OSMO',
+      imgUrl:
+        'https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.png',
+      votes: 3,
+      estimatedIncentive: valueView,
+    },
+    {
+      symbol: 'ATOM',
+      imgUrl:
+        'https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.png',
+      votes: 10,
+      estimatedIncentive: valueView,
+    },
+  ];
+
+  return (
+    <Card>
+      <div className='flex flex-col p-3 gap-4'>
+        <Text xxl color='text.primary'>
+          Current Voting Results
+        </Text>
+        <Density compact>
+          <div className='grid grid-cols-5 h-auto overflow-auto'>
+            <div className='grid grid-cols-subgrid col-span-5'>
+              <TableCell heading>Asset</TableCell>
+              <TableCell heading>Gauge Value</TableCell>
+              <TableCell heading>Casted Votes</TableCell>
+              <TableCell heading>Estimated Incentive</TableCell>
+              <TableCell heading>Vote</TableCell>
```

Comment:
todo: add sorting to the table with `useSortableTableHeaders`. Otherwise it'll be more work for the future